Python/Python.sublime-syntax

```diff
@@ -232,27 +403,30 @@ contexts:
     - match: '='
       scope: keyword.operator.assignment.python
       push:
+        - meta_scope: meta.function.default-value.python
         - match: '(?=[,)])'
           pop: true
         - include: expressions
     - match: ':'
-      scope: punctuation.separator.annotation.python
+      scope: punctuation.separator.annotation.parameter.python
       push:
+        - meta_scope: meta.function.annotation.parameter.python
```

Comment:
This is a similar scope stacking as just above where `meta.function` is stacked in the list.